index.js

```diff
@@ -179,12 +179,12 @@ module.exports = {
         }
       },
       afterPublish: {
-        async publishChildren(req, piece, options) {
+        async publishChildren(req, data) {
```

Comment:
Full docs for this event and its handler parameters: https://v3.docs.apostrophecms.org/reference/server-events.html#afterpublish